crml/staking/src/rewards/mod.rs

```diff
@@ -273,19 +273,19 @@ where
 		TransactionFeePot::<T>::get().saturating_add(Self::target_inflation_per_staking_era())
 	}
 
-	/// Calculate the next reward payout (accrued as of right now) for the given payee.
+	/// Calculate the next reward payout (accrued as of right now) for the given stash id.
 	fn payee_next_reward_payout(
-		payee: &Self::AccountId,
+		stash: &Self::AccountId,
 		validator_commission_stake_map: &[(Self::AccountId, Perbill, Exposure<Self::AccountId, Self::Balance>)],
 	) -> Self::Balance {
 		let mut payee_cut: Self::Balance = Zero::zero();
 
 		let (_, _, payouts) =
 			Self::calculate_payouts_filtered(validator_commission_stake_map, |validator, exposure| {
-				payee != validator && !exposure.others.iter().any(|x| &x.who == payee)
+				stash != validator && !exposure.others.iter().any(|x| &x.who == stash)
 			});
 		payouts.into_iter().for_each(|(account, payout)| {
-			if &account == payee {
+			if account == Self::payee(stash) {
```

Comment:
should query this once outside the loop and reuse the variable here

Reply:
Done